src/Query/LaravelExpressionProvider.php

```diff
@@ -140,7 +140,7 @@ public function onRelationalExpression($expressionType, $left, $right)
             case ExpressionType::NOTEQUAL:
                 return $this->_prepareBinaryExpression(self::NOT_EQUAL, $left, $right);
             default:
-                throw new \InvalidArgumentException('onArithmeticExpression');
+                throw new \InvalidArgumentException('onRelationalExpression');
```

Comment:
As this is the onRelationalExpression method, makes sense to update the exception message to match.